src/forces.jl

```diff
@@ -85,6 +85,42 @@ function accelerations(s::Simulation; parallel::Bool=true)
     return forces
 end
 
+function accelerations(s::Simulation, coords, coords_is, coords_js, atoms_is, atoms_js)
+    n_atoms = length(coords)
+    forces = zero(coords)
+
+    for inter in values(s.general_inters)
+        if inter.nl_only
+            forces -= reshape(sum(force.((inter,), coords_is, coords_js, atoms_is, atoms_js,
+                                            s.box_size), dims=2), n_atoms)
+        else
+            forces -= reshape(sum(force.((inter,), coords_is, coords_js, atoms_is, atoms_js,
+                                            s.box_size), dims=2), n_atoms)
+        end
```

Comment:
Is the if necessary `if` both branches do the same thing?

Reply:
Not currently, I intend to replace this with something that actually uses the neighbours though.

Reply:
If we give the coords and the atoms to the function, isn't that equivalent to using a neighbor list?
The caller of the function could just use a part of the `i`s and `j`s and here we could just keep the implementation generic.

Reply:
There would have to be some way of summing the forces afterwards though, at the minute the 2D broadcast returns things in the correct shape for summing. I did think of doing something with a `SparseVector`, similar to the specific interaction case.